eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/configuration/CreateDeprecatedReleaseStepConfigurationTest.java

```diff
@@ -0,0 +1,124 @@
+import com.lordofthejars.nosqlunit.mongodb.MongoDbRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.batch.core.BatchStatus;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.test.JobLauncherTestUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import uk.ac.ebi.eva.accession.release.io.DeprecatedVariantAccessionWriter;
+import uk.ac.ebi.eva.accession.release.parameters.InputParameters;
+import uk.ac.ebi.eva.accession.release.parameters.ReportPathResolver;
+import uk.ac.ebi.eva.accession.release.test.configuration.BatchTestConfiguration;
+import uk.ac.ebi.eva.accession.release.test.configuration.MongoTestConfiguration;
+import uk.ac.ebi.eva.accession.release.test.rule.FixSpringMongoDbRule;
+import uk.ac.ebi.eva.commons.core.utils.FileUtils;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static uk.ac.ebi.eva.accession.release.configuration.BeanNames.RELEASE_MAPPED_DEPRECATED_VARIANTS_STEP;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = {BatchTestConfiguration.class, MongoTestConfiguration.class})
+@UsingDataSet(locations = {
+        "/test-data/dbsnpClusteredVariantOperationEntity.json",
+        "/test-data/dbsnpSubmittedVariantEntity.json"})
+@TestPropertySource("classpath:application.properties")
+public class CreateDeprecatedReleaseStepConfigurationTest {
+
+    private static final String TEST_DB = "test-db";
+
+    private static final long EXPECTED_LINES = 1;
+
+    @Autowired
+    private JobLauncherTestUtils jobLauncherTestUtils;
+
+    @Autowired
+    private InputParameters inputParameters;
+
+    //Required by nosql-unit
+    @Autowired
+    private ApplicationContext applicationContext;
+
+    @Rule
+    public MongoDbRule mongoDbRule = new FixSpringMongoDbRule(
+            MongoDbConfigurationBuilder.mongoDb().databaseName(TEST_DB).build());
+
+    @Test
+    public void contextLoads() {
+
+    }
+
+    @Test
+    public void basicStepCompletion() throws Exception {
+        assertStepExecutesAndCompletes();
+    }
+
+    private void assertStepExecutesAndCompletes() {
+        JobExecution jobExecution = jobLauncherTestUtils.launchStep(RELEASE_MAPPED_DEPRECATED_VARIANTS_STEP);
+        assertEquals(BatchStatus.COMPLETED, jobExecution.getStatus());
+    }
+
+    @Test
+    public void variantsWritten() throws Exception {
+        assertStepExecutesAndCompletes();
+        long numVariantsInRelease = FileUtils.countNonCommentLines(new FileInputStream(getDeprecatedReleaseFile()));
+        assertEquals(EXPECTED_LINES, numVariantsInRelease);
```

Comment:
Just as a matter of principle, shall we do 2 lines for this test (as a guard against file/list overwrites etc.,)?